Build.cmd

```diff
@@ -104,13 +104,20 @@ goto :DoBuild
 
 :DoBuild
 
+"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -Latest -requires Microsoft.Component.MSBuild -find MSBuild\**\Bin\MSBuild.exe > %TEMP%\msbuildpath.txt
+
+set /p MSBUILDPATH=<%TEMP%\msbuildpath.txt
```

Comment:
I have some changes I'd like to propose to make this not use a temp file -- do you mind if I push to your remote?

Reply:
I pushed it as [user/jevansaks/azchohfi_fixSxSBuild](https://github.com/Microsoft/microsoft-ui-xaml/commit/e2dcd41493d87b1856588a1f0fcf93abe0b17ff1) if you want to take a look.

Reply:
I copied that from DevCmd.cmd:
https://github.com/Microsoft/microsoft-ui-xaml/blob/4a94f1e5e16f70ab4337e1b7bc6e9dfbf041b571/DevCmd.cmd#L9
But I'm happy to accept your change.

Reply:
Ah, well then you'd be fixing my mistake. Want to apply the same change to DevCmd? (It's ok if you don't)